source/manage/logging.rst

```diff
@@ -185,7 +185,7 @@ Define advanced log output
                     "MaxBackups": 10,
                     "MaxSizeMB": 500
```

Comment:
@wiggin77 Is that correct? The docs in http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/7101/manage/logging.html#file-target-configuration-options state that it's `max_size`.

Reply:
The linked docs are correct:
```go
type FileOptions struct {
	// Filename is the file to write logs to.  Backup log files will be retained
	// in the same directory.  It uses <processname>-lumberjack.log in
	// os.TempDir() if empty.
	Filename string `json:"filename"`

	// MaxSize is the maximum size in megabytes of the log file before it gets
	// rotated. It defaults to 100 megabytes.
	MaxSize int `json:"max_size"`

	// MaxAge is the maximum number of days to retain old log files based on the
	// timestamp encoded in their filename.  Note that a day is defined as 24
	// hours and may not exactly correspond to calendar days due to daylight
	// savings, leap seconds, etc. The default is not to remove old log files
	// based on age.
	MaxAge int `json:"max_age"`

	// MaxBackups is the maximum number of old log files to retain.  The default
	// is to retain all old log files (though MaxAge may still cause them to get
	// deleted.)
	MaxBackups int `json:"max_backups"`

	// Compress determines if the rotated log files should be compressed
	// using gzip. The default is not to perform compression.
	Compress bool `json:"compress"`
}
```

Reply:
But the examples are still wrong, right? They are using `MaxSizeMB`, but should be using either `max_backups` or `MaxBackups`.

Reply:
Opened https://github.com/mattermost/docs/pull/7109